src/test/java/seedu/address/model/person/EmailTest.java

```diff
@@ -65,4 +65,14 @@ public void isValidEmail() {
         assertTrue(Email.isValidEmail("if.you.dream.it_you.can.do.it@example.com")); // long local part
         assertTrue(Email.isValidEmail("e1234567@u.nus.edu")); // more than one period in domain
     }
+
+    @Test
+    void testHashCode() {
+        Email firstEmail = new Email("PeterJack_1190@example.com");
+        Email firstEmailCopy = new Email("PeterJack_1190@example.com");
+        Email secondEmail = new Email("peterjack@example.com");
+
+        assertTrue(firstEmail.hashCode() == firstEmailCopy.hashCode());
```

Comment:
can add a test case to check if the hashCode is the same across the same object like `firstEmail.hashcode() == firstEmail.hashcode()` to ensure the hashCode function is not generated randomly.